python/.env.example

```
@@ -0,0 +1,155 @@
+# ============================================================================
+# OPTIONAL - Crawling Configuration
+# ============================================================================
+
+# Maximum pages to crawl per source
+MAX_PAGES_PER_CRAWL=100
+
+# Crawl delay between requests (milliseconds)
+CRAWL_DELAY_MS=1000
+
+# User agent for web crawling
+USER_AGENT=Archon/1.0 (+https://github.com/yourusername/archon)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)